consul_exporter.go

```diff
@@ -86,6 +86,15 @@ func NewExporter(uri string) *Exporter {
 			[]string{"service", "node"},
 		),
 
+		nodeChecks: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: namespace,
+				Name:      "agent_check",
+				Help:      "Is this check passing on this node?",
```

Comment:
You could even go crazy and encode more states than 0 or 1. I don't know Consul enough to know if that's useful.

The reason putting it into the value is nicer:

- otherwise you'll either end up with many short-lived time series (any unique combination of labels results in a new time series)
- if you want to do stuff like "how often has this time series been 0 vs. 1" in a given period of time, that works much better with a single series